webdriver-spec.html

```diff
@@ -3228,7 +3228,7 @@ <h2 id=element-displayedness>Element Displayedness</h2>
   </section>
 </section>
 <section>
-  <h2>Executing Javascript</h2>
+  <h2>Executing Script</h2>
   <div class="note">
```

Comment:
Oh, sorry, I misunderstood GH because the highlighted lines and the lines you commented on were not the same.